_layouts/compress.html

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
We don't need this, we can just put it behind CloudFlare which can minify everything for us